enterprise/cmd/worker/internal/embeddings/repo/handler.go

```diff
@@ -34,9 +34,11 @@ const MAX_FILE_SIZE = 1000000 // 1MB
 
 // The threshold to embed the entire file is slightly larger than the chunk threshold to
 // avoid splitting small files unnecessarily.
-const EMBED_ENTIRE_FILE_TOKENS_THRESHOLD = 384
-const EMBEDDING_CHUNK_TOKENS_THRESHOLD = 256
-const EMBEDDING_CHUNK_EARLY_SPLIT_TOKENS_THRESHOLD = EMBEDDING_CHUNK_TOKENS_THRESHOLD - 32
+const (
+	EMBED_ENTIRE_FILE_TOKENS_THRESHOLD           = 384
+	EMBEDDING_CHUNK_TOKENS_THRESHOLD             = 256
+	EMBEDDING_CHUNK_EARLY_SPLIT_TOKENS_THRESHOLD = EMBEDDING_CHUNK_TOKENS_THRESHOLD - 32
```

Comment:
nit: if you touch this chunk, we might as well make it PascalCase :) (if it has to be exported)
```suggestion
	EmbedEntireFileTokensThreshold          = 384
	EmbeddingChunkTokensThreshold           = 256
	EmbeddingChunkEarlySplitTokensThreshold = EmbeddingChunkTokensThreshold - 32
```